iree_tests/README.md

```diff
@@ -1,5 +1,39 @@
 # IREE Tests
 
+## Quickstart instructions
+
+```
+#!/bin/bash
+set -e
+
+cd ~/SHARK-TestSuite/
+
+# Create and activate a virtual environment
+python3 -m venv .venv
+source .venv/bin/activate
+
+# Install required packages
+pip install -r iree_tests/requirements.txt
+pip install ./common_tools/
+
+# Optional: Install IREE compiler and runtime if you don't want to use a local copy
+# pip install --find-links https://iree.dev/pip-release-links.html iree-compiler iree-runtime --upgrade
+
+# Install Git LFS
+git lfs install
```

Comment:
It helps to run this before cloning the repo IIRC. I'm not sure if installing git lfs will automatically fetch blobs if you had previously cloned/fetched without git lfs.